easybuild/easyconfigs/b/BBMap/BBMap-35.82-foss-2015b-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '35.82'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz' ]
+
+toolchain = {'name': 'foss', 'version': '2015b'}
```

Comment:
@nathanhaigh: at the risk of coming across as an annoying brat: `source_urls` and `sources` go *below* `toolchain` usually...